### PR TITLE
Chore/merge-v4-dev-@261418

### PR DESCRIPTION
**Pull request by julien-deramond**
- https://github.com/twbs/bootstrap/commit/5299abeeab344f33cb311badc259ed1380fad130: Already done in Boosted
- https://github.com/twbs/bootstrap/commit/3b63d509dee06f012ec8b53b7ae50255dc297e2d: Already done in Boosted
- [x] https://github.com/twbs/bootstrap/commit/bae047fceee66f4180c69c2f0da709e470e9b504
- [x] https://github.com/twbs/bootstrap/commit/0090af6a9cf19c2aba44dd233230e1371bef1ab9
- [x] https://github.com/twbs/bootstrap/commit/7613953998dac491b4ade617465b6b1c4f963e41
- [x] https://github.com/twbs/bootstrap/commit/d36b1a46cf17ed5076bc86799936de2451ebc7f0
- [x] https://github.com/twbs/bootstrap/commit/d9481ee0543935bd4065d44b030b782e3fcce510
- [x] https://github.com/twbs/bootstrap/commit/6a500849de6d00950f32fcd5c949657e815039c1
- [x] https://github.com/twbs/bootstrap/commit/e4dc2e91a5fd555595f6a16d58699917ab3e24e4
- [x] https://github.com/twbs/bootstrap/commit/7d57d9a68bbc2bc703fbf2c66fd2ad2546d67557
- https://github.com/twbs/bootstrap/commit/c1b42e3538b7c67f61b728ad5bafee6e63f1706b: Already done in Boosted
- [x] https://github.com/twbs/bootstrap/commit/c043a75d5317132a84ad1eabdde2b91dde23c03e
- [x] https://github.com/twbs/bootstrap/commit/261b418798481ed2d50a60a3aa95331ce99784d3

### CODE REVIEW

File: site/content/docs/4.6/components/navs.md [hunk 1]
```diff
@@ -510,10 +510,10 @@ And with vertical pills.
...
     <div class="nav flex-column nav-pills" id="v-pills-tab" role="tablist" aria-orientation="vertical">
-      <a class="nav-link active" id="v-pills-home-tab" data-toggle="pill" href="#v-pills-home" role="tab" aria-controls="v-pills-home" aria-selected="true">Home</a>
-      <a class="nav-link" id="v-pills-profile-tab" data-toggle="pill" href="#v-pills-profile" role="tab" aria-controls="v-pills-profile" aria-selected="false">Profile</a>
-      <a class="nav-link" id="v-pills-messages-tab" data-toggle="pill" href="#v-pills-messages" role="tab" aria-controls="v-pills-messages" aria-selected="false">Messages</a>
-      <a class="nav-link" id="v-pills-settings-tab" data-toggle="pill" href="#v-pills-settings" role="tab" aria-controls="v-pills-settings" aria-selected="false">Settings</a>
+      <button class="nav-link active" id="v-pills-home-tab" data-toggle="pill" data-target="#v-pills-home" type="button" role="tab" aria-controls="v-pills-home" aria-selected="true">Home</button>
+      <button class="nav-link" id="v-pills-profile-tab" data-toggle="pill" data-target="#v-pills-profile" type="button" role="tab" aria-controls="v-pills-profile" aria-selected="false">Profile</buttona>
```
Comment (julien-deramond) [inline site/content/docs/4.6/components/navs.md:514]: `</buttona>` needs to be fixed in Bootstrap

File: site/content/docs/4.6/components/navs.md [hunk 2]
```diff
@@ -396,9 +396,9 @@ To help fit your needs, this works with `<ul>`-based markup, as shown above, or
 <div class="bd-example bd-example-tabs">
```
Comment (louismaximepiton) [inline site/content/docs/4.6/components/navs.md:396]: (out from this PR) This example is broken using keyboard -> Not accessible and not the only one

**Review by louismaximepiton — COMMENTED** [review]
Few things for Boosted and the main things to Bs.

Could maybe be tackled in another PR but the javascript behavior on tabs seems to be completely broken on our side. The `_dataApiKeydownHandler` seems to not handle every case

**Review by louismaximepiton — APPROVED** [review]

**Comment by julien-deramond** [comment]
I let you tackle the things directly in Boostrap and create an issue for the problems not coming from this PR. Thanks for the review.

File: site/content/docs/4.6/components/navs.md (hunk 2 above)
Comment (louismaximepiton) [reply site/content/docs/4.6/components/navs.md:396]: Tackled by #1294

File: js/tests/unit/tab.js [hunk 3]
```diff
@@ -79,48 +95,48 @@ $(function () {
   QUnit.test('should activate element by tab id in ordered list', function (assert) {
     assert.expect(2)
     var pillsHTML = '<ol class="nav nav-pills">' +
-        '<li><a href="#home">Home</a></li>' +
-        '<li><a href="#profile">Profile</a></li>' +
+        '<li><button type="button" data-target="#home" role="tab">Home</button></li>' +
+        '<li><button type="button" href="#profile" role="tab">Profile</button></li>' +
```
[the remark this answers is not in the document]
Comment (louismaximepiton) [reply js/tests/unit/tab.js:99]: See https://github.com/twbs/bootstrap/pull/36485/files

File: js/tests/unit/tab.js [hunk 4]
```diff
@@ -79,48 +95,48 @@ $(function () {
   QUnit.test('should activate element by tab id in ordered list', function (assert) {
     assert.expect(2)
     var pillsHTML = '<ol class="nav nav-pills">' +
```
[the remark this answers is not in the document]
Comment (louismaximepiton) [reply js/tests/unit/tab.js:97]: See https://github.com/twbs/bootstrap/pull/36485/files

File: js/tests/unit/tab.js [hunk 5]
```diff
@@ -79,48 +95,48 @@ $(function () {
...
     assert.strictEqual($('#qunit-fixture').find('.active').attr('id'), 'home')
   })
 
   QUnit.test('should activate element by tab id in nav list', function (assert) {
     assert.expect(2)
     var tabsHTML = '<nav class="nav">' +
-                      '<a href="#home">Home</a>' +
-                      '<a href="#profile">Profile</a>' +
+                      '<button type="button" data-target="#home" role="tab">Home</button>' +
+                      '<button type="button" data-target="#profile" role="tab">Profile</a>' +
```
[the remark this answers is not in the document]
Comment (louismaximepiton) [reply js/tests/unit/tab.js:115]: See https://github.com/twbs/bootstrap/pull/36485/files

File: js/tests/unit/tab.js [hunk 6]
```diff
@@ -517,10 +510,10 @@ $(function () {
     var html = [
       '<ul class="nav nav-tabs" role="tablist">',
       '  <li class="nav-item" role="presentation">',
-      '    <a class="nav-link nav-tab" href="#home" role="tab" data-toggle="tab">Home</a>',
+      '     <button type="button" class="nav-link nav-tab" data-target="#home" role="tab" data-toggle="tab">Home</button>',
```
[the remark this answers is not in the document]
Comment (louismaximepiton) [reply js/tests/unit/tab.js:513]: See https://github.com/twbs/bootstrap/pull/36485/files

File: js/tests/visual/tab.html [hunk 7]
```diff
@@ -196,10 +193,10 @@ <h4>Tabs with list-group (with fade)</h4>
...
-            <a class="list-group-item list-group-item-action active" id="list-home-list" data-toggle="tab" href="#list-home" role="tab" aria-controls="list-home">Home</a>
-            <a class="list-group-item list-group-item-action" id="list-profile-list" data-toggle="tab" href="#list-profile" role="tab" aria-controls="list-profile">Profile</a>
-            <a class="list-group-item list-group-item-action" id="list-messages-list" data-toggle="tab" href="#list-messages" role="tab" aria-controls="list-messages">Messages</a>
-            <a class="list-group-item list-group-item-action" id="list-settings-list" data-toggle="tab" href="#list-settings" role="tab" aria-controls="list-settings">Settings</a>
+            <button type="button" class="list-group-item list-group-item-action active" id="list-home-list" data-toggle="tab" data-target="#list-home" role="tab" aria-controls="list-home" aria-selected="true">Home</button>
+            <button type="button"  class="list-group-item list-group-item-action" id="list-profile-list" data-toggle="tab" data-target="#list-profile" role="tab" aria-controls="list-profile">Profile</button>
```
[the remark this answers is not in the document]
Comment (louismaximepiton) [reply js/tests/visual/tab.html:197]: See https://github.com/twbs/bootstrap/pull/36485/files

File: js/tests/unit/tab.js [hunk 8]
```diff
@@ -211,44 +224,44 @@ $(function () {
     assert.expect(2)
     var done = assert.async()
 
-    var tabsHTML = '<ul class="nav">' +
-        '<li><a href="#home">Home</a></li>' +
-        '<li><a href="#profile">Profile</a></li>' +
+    var tabsHTML = '<ul class="nav" role="tablist">' +
+        '<li><button type="button" data-target="#home" role="tab">Home</button></li>' +
+        '<li><button type="button" data-target="#profile">Profile</button></li>' +
```
[the remark this answers is not in the document]
Comment (louismaximepiton) [reply js/tests/unit/tab.js:229]: See https://github.com/twbs/bootstrap/pull/36485/files

**Pull request merged**